fe/fe-core/src/main/java/com/starrocks/connector/Connector.java

```diff
@@ -31,4 +33,8 @@ public interface Connector {
      * have been returned from the connector.
      */
     default void shutdown() {}
+
+    default CloudConfiguration getCloudConfiguration() {
```

Comment:
add TODO.  indicating that this is a temp interface to solve which issue. and you will refactor ASAP in future.